cub/test/test_util.h

```diff
@@ -991,6 +991,31 @@ struct NumericTraits<TestFoo>
 };
 CUB_NAMESPACE_END
 
+_LIBCUDACXX_BEGIN_NAMESPACE_STD
+template <>
+class numeric_limits<TestFoo>
+{
+private:
```

Comment:
That seems a bit strict? why isn't this public?

Reply:
Nice one, thx!